fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/NavigationBar.kt

```diff
@@ -68,8 +88,10 @@ fun NavigationBar(
                         }
                     }
                     ItemType.TAB_COUNTER -> {
-                        IconButton(onClick = {}) {
-                            TabsIcon(item = it, tabsCount = 0)
+                        CompositionLocalProvider(LocalContentColor provides FirefoxTheme.colors.iconPrimary) {
+                            IconButton(onClick = {}) {
```

Comment:
I'm not familiar with the tasks yall have already planned out for next steps. Will this click lambda eventually be used? Will the `IconButton` be used to trigger the showing of the Tabs Tray?

Reply:
I am not sure yet what's the best approach to implement clicks. Do you have any suggestions?

Reply:
An `IconButton` should be fine, I was just asking to clarify if the click was actually going to be consumed, and, if so, what the follow-up was going to be.

Reply:
just to close the loop, synced with @MozillaNoah about it; the tap will be triggering the same behaviour we currently have in the toolbar when the tab button is clicked.